### PR TITLE
Rewrite cef gb to structuredataspider, however this requires a proxy

### PR DESCRIPTION
**Pull request by CloCkWeRX**

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
Can't quite test it locally

File: locations/spiders/cef_gb.py [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
-    def parse(self, response, **kwargs):
-        ld = response.xpath('//script[@type="application/ld+json"]/text()').get()
-        ld = ld.replace("&quot;", '"')
-        ld_item = json.loads(ld)
-
-        item = LinkedDataParser.parse_ld(ld_item)
-        item["ref"] = response.url
-
+    def post_process_item(self, item, response, ld_data):
         extract_google_position(item, response)
```
Comment (CloCkWeRX) [inline locations/spiders/cef_gb.py:20]:
```suggestion
        extract_google_position(ld_data, response)
```

Comment (CloCkWeRX) [inline locations/spiders/cef_gb.py:20]: Odd this bit didn't work.

Comment (CloCkWeRX) [reply locations/spiders/cef_gb.py:20]: https://github.com/alltheplaces/alltheplaces/issues/10399

Comment (iandees) [reply locations/spiders/cef_gb.py:20]: This function will only look in links, not inside of JSON.T

**Pull request merged**